env0/configuration_variable.go

```diff
@@ -0,0 +1,143 @@
+package env0
```

Comment:
moved these functions to here from env0/resource_environment.go
(These are mostly commom functions).